### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by zglismll**

### CODE REVIEW

**Comment by zglismll** [comment]
@dla-kirito

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <html>
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+  <meta http-equiv="Contype" content="text/html; charset=utf-8"/>
```
Comment (zglismll) [inline index.html:5]: 啦啦啦啦啦啦啦啦绿绿绿

Comment (dla-kirito) [reply index.html:5]: test reply

Comment (dla-kirito) [reply index.html:5]: > 啦啦啦啦啦啦啦啦绿绿绿

reply

File: index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <html>
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
```
Comment (zglismll) [inline index.html:5]: @dla-kirito

File: index.html (hunk 1 above)
Comment (dla-kirito) [inline index.html:5]: test

Comment (dla-kirito) [reply index.html]: 222

**Review by dla-kirito — APPROVED** [review]

**Comment by fi3ework** [comment]
> test
bad dog

**Review by dla-kirito — APPROVED** [review]

**Comment by fi3ework** [comment]
gdx

**Pull request closed without merging**